src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -494,6 +494,17 @@ public PrivilegeSummary getBookingIEPSummary(@PathVariable("bookingId") @ApiPara
 
     }
 
+    @ApiResponses({
+        @ApiResponse(code = 400, message = "Invalid request.", response = ErrorResponse.class),
+        @ApiResponse(code = 404, message = "Requested resource not found.", response = ErrorResponse.class),
+        @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class)})
+    @ApiOperation(value = "Prisoners IEP (Incentives & Earned Privileges) summary for a list of booking IDs", notes = "Must have prisoner in users caseload access data")
+    @PostMapping("/iepSummary")
+    public Collection<PrivilegeSummary> getBookingIEPSummaryDetailForBookingIds(@NotNull @RequestBody @ApiParam(value = "The booking ids of prisoners", required = true) final List<Long> bookings) {
```

Comment:
I'm surpised you are not returning a map, I would have thought any client would want the summary mapped to the bookingId supplied

Reply:
Very true! That's a good idea